packages/hydrogen-edge-adapter/src/index.ts

```diff
@@ -0,0 +1,16 @@
+export const netlifyEdgeConfig = {
+  path: '/*',
+
+  // Allow Remix to handle caching Edge Functions
+  // https://docs.netlify.com/edge-functions/optional-configuration/#configure-an-edge-function-for-caching
+  cache: 'manual',
+
+  // Let the CDN handle requests for static assets, i.e. ^/_assets/*$
+  // Add other exclusions here, e.g. "^/api/*$" for custom Netlify functions or custom Netlify Edge Functions
+  excludedPath: ['/build/*'],
+
+  // Use built-in Remix error handling
```

Comment:
Nice! We didn't have this when I first set up the edge adapter. Have you tested it when something fails? I'm going to put up a PR for the edge adapter to add this as well.

Reply:
https://docs.netlify.com/edge-functions/declarations/#declare-edge-functions-inline has the docs for this piece specifically.

What do you mean specifically when you’re referring to a failure? https://resonant-scone-e069de.netlify.app/ has an example app with this code applied, do you mean like a 404 in the `/build/` directory?

Reply:
Ah I’ve completely misunderstood the context of your comment 😅. You’re talking about `onError` right? 🫣